docs/rules/periphery/p046-mf_dotdash.csv

```diff
@@ -23,3 +23,4 @@ Name,Description,Flags,Value
 (mf.22),Min spacing between fuse_contact to fuse_contact,,1.960
 (mf.23),Spacing (no overlapping) between fuse center and Metal4,,N/A
 (mf.24),Spacing (no overlapping) between fuse center and Metal5,,3.300
+(mf.Section G2b: Rules for HV devices),,,

```

Comment:
Something is going wrong with this rule here too. Let's log a bug to fix it?

Reply:
Just looked into this one -- I think it's just because of the comma being in the wrong place for that one row.

![image](https://user-images.githubusercontent.com/855559/94366476-2b868100-009e-11eb-8245-98130c0d3b27.png)

(notice how the top examples put it in the second column, but the example you called out here on the bottom is in the first column).  So, it's probably just human error. Will fix.

Reply:
This was fixed in commit 123ad90